packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -137,154 +163,180 @@ export type TileProps = {
  *  isNew={false}
  * />
  */
-export const Tile = (props: TileProps) => {
-  const {
-    className,
-    name,
-    description,
-    metadata,
-    thumbnail,
-    buttons,
-    leftIcon,
-    rightIcon,
-    badge,
-    isNew,
-    isSelected,
-    moreOptions,
-    variant = 'default',
-    children,
-    isActionable,
-    status,
-    isLoading = false,
-    isDisabled = false,
-    ...rest
-  } = props;
+export const Tile = Object.assign(
+  (props: TileProps) => {
+    const {
+      className,
+      name,
+      description,
+      metadata,
+      thumbnail,
+      buttons,
+      leftIcon,
+      rightIcon,
+      badge,
+      isNew,
+      isSelected,
+      moreOptions,
+      variant = 'default',
+      children,
+      isActionable,
+      status,
+      isLoading = false,
+      isDisabled = false,
+      onClick,
+      ...rest
+    } = props;
 
-  useTheme();
+    useTheme();
 
-  const [isMenuVisible, setIsMenuVisible] = React.useState(false);
-  const showMenu = React.useCallback(() => setIsMenuVisible(true), []);
-  const hideMenu = React.useCallback(() => setIsMenuVisible(false), []);
+    const [isMenuVisible, setIsMenuVisible] = React.useState(false);
+    const showMenu = React.useCallback(() => setIsMenuVisible(true), []);
+    const hideMenu = React.useCallback(() => setIsMenuVisible(false), []);
+    const [localActionable, setLocalActionable] = React.useState(isActionable);
 
-  const tileName = (
-    <div className='iui-tile-name'>
-      <TitleIcon
-        isLoading={isLoading}
-        isSelected={isSelected}
-        isNew={isNew}
-        status={status}
-      />
-
-      <span className='iui-tile-name-label'>{name}</span>
-    </div>
-  );
+    React.useEffect(() => {
+      console.log('tile useeffect');
+      setLocalActionable(isActionable);
```

Comment:
this use effect is not needed.

you can derive a value like this:
```js
const isActionable = isActionableProp ?? localActionable;
```

see https://beta.reactjs.org/learn/you-might-not-need-an-effect#updating-state-based-on-props-or-state

Reply:
Removed useEffect